src/changelog/comment_changes.rs

```diff
@@ -50,6 +50,10 @@ pub struct CommentChanges {
   #[arg(long, short = 'C')]
   fallback_category: Option<String>,
 
+  /// Whether to enforce the fragment creation.
+  #[arg(long, short = 'F')]
+  force: bool,
```

Comment:
The option to trigger the search for changes in the opposite message part is named `--force`, abbreviated by `-F`.